WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -116,21 +121,38 @@ struct OrderFormPresentationWrapper: View {
                           presentProductSelector: presentProductSelector)
             }, secondaryView: { isShowingProductSelector in
                 if let productSelectorViewModel = viewModel.productSelectorViewModel {
-                    ProductSelectorView(configuration: ProductSelectorView.Configuration.splitViewAddProductToOrder(),
+                    ProductSelectorView(configuration: productSelectorConfiguration,
```

Comment:
As mentioned in Slack, you couldn't use the `adaptiveModalContainerPresentationStyle` environment key here, because it's injected to the environment by the `AdaptiveModalContainer`, and this (`secondaryView` ViewBuilder) is _not_ a descendent of the container.

It's subtle – the `ProductSelectorView` itself does use the environment key, because the view that this builder renders _is_ a descendent of the container.

Perhaps the environment key isn't set in the right place... but I don't see how we'd do it further up the heirarachy. Given that it's really just a wrapper around the system-provided `horizontalSizeClass` key, we should probably remove it entirely and just use that, combined with a feature flag check. (At the moment, this is only set to `.sideBySide` if the size class is regular _and_ the adaptive modal container is in use, i.e. `splitViewAddProductToOrder` is true)

I'll take a look at it later this week, as I think the custom env key is just adding complexity at the moment. In future, it could be useful (e.g. if we give users control to override the mode that's used) but that's speculative.